src/ORM/Connect/DBSchemaManager.php

```diff
@@ -529,6 +533,28 @@ public function requireIndex($table, $index, $spec)
         }
     }
 
+    public function dontRequireIndex(string $table, string $index): void
```

Comment:
The name of this method is consistent with other similar methods (`dontRequireTable`, `dontRequireField`)